vortex-datafusion/src/persistent/opener.rs

```diff
@@ -44,9 +44,29 @@ impl VortexFileOpener {
             .as_ref()
             // If we cannot convert an expr to a vortex expr, we run no filter, since datafusion
             // will rerun the filter expression anyway.
-            .and_then(|expr| convert_expr_to_vortex(expr.clone()).ok())
-            .map(|expr| simplify_typed(expr, dtype))
+            .map(|expr| {
```

Comment:
I guess the matches the `can_be_pushed_down` function, so it works, we need to unify this.